### PR TITLE
Adding outside furniture

### PR DESCRIPTION
**Pull request by arijust**
#### Summary

```SUMMARY: Content "Adds street light, traffic lights and utility pole."```
#### Purpose of change

Helps #28187
#### Additional context

I still need to add items for salvage, but I could use ideas with what add to these.
Main look_like for these will be utility pole.

### CODE REVIEW

File: data/json/furniture_and_terrain/terrain-manufactured.json [hunk 1]
```diff
@@ -1437,5 +1437,78 @@
...
+        { "item": "plastic_chunk", "count": [ 2, 4 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_traffic_light",
+    "name": "traffic light",
+    "looks_like": "t_utility_pole",
+    "description": "Signalling device positioned at road intersections and pedestrian crossings to control flows of traffic, but it's useless without electricity..",
```
Comment (AMurkin) [inline data/json/furniture_and_terrain/terrain-manufactured.json]:
```suggestion
    "description": "Signalling device positioned at road intersections and pedestrian crossings to control flows of traffic, but it's useless without electricity.",
```

File: data/json/furniture_and_terrain/terrain-manufactured.json [hunk 2]
```diff
@@ -1437,5 +1437,78 @@
...
     "bash": { "sound": "thump", "ter_set": "t_dirt", "str_min": 50, "str_max": 100, "str_min_supported": 100 }
+  },
+  {
+    "type": "terrain",
+    "id": "t_street_light",
+    "name": "street light",
+    "looks_like": "t_utility_pole",
+    "description": "Raised source of light which used to illuminate streets and their surrounding area, but it's useless without electricity.",
+    "symbol": "T",
+    "color": "grey",
```
Comment (AMurkin) [inline data/json/furniture_and_terrain/terrain-manufactured.json]: Could be `light_gray` or `dark_gray`.

File: data/json/furniture_and_terrain/terrain-manufactured.json [hunk 3]
```diff
@@ -1437,5 +1437,78 @@
...
+      "str_min": 30,
+      "str_max": 100,
+      "sound": "metal screeching!",
+      "sound_fail": "clang!",
+      "ter_set": "t_thconc_floor",
+      "items": [
+        { "item": "steel_lump", "count": [ 0, 2 ] },
+        { "item": "steel_chunk", "count": [ 1, 4 ] },
+        { "item": "scrap", "count": [ 3, 12 ] },
+        { "item": "plastic_chunk", "count": [ 2, 4 ] }
```
Comment (Rail-Runner) [inline data/json/furniture_and_terrain/terrain-manufactured.json]: Could probably drop some suitable electronic components as well if broken, I think. Same for traffic light and utility pole.

**Comment by snipercup** [comment]
I tested it:

image


I cannot move onto the poles (as expected)
cannot deconstruct
can bash and it breaks down. Wood axe is not sufficient, used sledge hammer

Cannot place trap:
image



Was wondering if this should be furniture instead of terrain?

File: data/json/furniture_and_terrain/furniture-terrains.json [hunk 4]
```diff
@@ -670,5 +670,84 @@
...
+        { "item": "plastic_chunk", "count": [ 2, 4 ] }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_traffic_light",
+    "name": "traffic light",
+    "looks_like": "f_utility_pole",
+    "description": "Signalling device positioned at road intersections and pedestrian crossings to control flows of traffic, but it's useless without electricity.",
```
Comment (Night-Pryanik) [inline data/json/furniture_and_terrain/furniture-terrains.json]: American English.
```suggestion
    "description": "Signaling device positioned at road intersections and pedestrian crossings to control flows of traffic, but it's useless without electricity.",
```

**Comment by Night-Pryanik** [comment]
You need to spawn these somewhere. Otherwise they are a dead weight.

**Comment by arijust** [comment]
I have no idea where add traffic lights.

**Pull request merged**